GraphDefinition/GraphDefinition-Genomics-Test-Result.json

```diff
@@ -0,0 +1,181 @@
+{
+  "resourceType": "GraphDefinition",
+  "id": "GraphDefinition-Genomics-Test-Result",
```

Comment:
For conformance to all other assets that have the ID as part of their canonical URL, the ID should be all lower case.